features/preferences/impl/src/main/kotlin/io/element/android/features/preferences/impl/advanced/AdvancedSettingsPresenter.kt

```diff
@@ -39,11 +38,9 @@ class AdvancedSettingsPresenter @Inject constructor(
         val doesCompressMedia by remember {
             sessionPreferencesStore.doesCompressMedia()
         }.collectAsState(initial = true)
-        val theme by remember {
+        val theme = remember {
```

Comment:
Small neat: Why not keeping the `by` pattern?

Reply:
Using `by` on a state will make the `Composable` function to be recomposed each time the value changes, defeating the purpose of the `derivedState`